Python/pylifecycle.c

```diff
@@ -1529,11 +1552,16 @@ finalize_modules(PyThreadState *tstate)
     // Remove all modules from sys.modules, hoping that garbage collection
     // can reclaim most of them: set all sys.modules values to None.
     //
-    // We prepare a list which will receive (name, weakref) tuples of
-    // modules when they are removed from sys.modules.  The name is used
-    // for diagnosis messages (in verbose mode), while the weakref helps
-    // detect those modules which have been held alive.
-    PyObject *weaklist = finalize_remove_modules(modules, verbose);
+    // This prepares two lists, the user defined list of modules as well
+    // as stdlib list of modules. The user modules will be destroyed first in
```

Comment:
```suggestion
    // as the stdlib list of modules. The stdlib modules will be destroyed after all user modules in
```
(And probably reflow.)